scripts/train.py

```diff
@@ -0,0 +1,44 @@
+import os
+import click
+import pickle
+import sys
+from sklearn.neural_network import MLPClassifier
+from pkspace.utils.loaders import PKSpaceLoader, PKLotLoader
+from pkspace.utils import trainer
+
+
+@click.command()
+@click.option('--loader', '-l', type=click.Choice(['PKLot', 'PKSpace']),
+              default='PKSpace', help='Loader used to load dataset')
+@click.option('--model_type', '-mt', type=click.Choice(['MLP']), default='MLP',
+              help='Type of model to be trained')
+@click.option('--model_path', '-pm', type=click.Path(exists=True),
+              default=None,
+              help='Path to trained model, to be used as a base in training')
+@click.argument('dataset_dir', type=click.Path(exists=True))
+@click.option('--output', '-o', default='out.pkl',
+              help='Path to output file for trained model')
+def train(loader, model_type, model_path, dataset_dir, output):
+    if not os.path.isdir(dataset_dir):
+        sys.stderr.write('{} is not a directory'.format(dataset_dir))
+        sys.exit(1)
+
+    if loader == 'PKSpace':
+        loader = PKSpaceLoader()
+    elif loader == 'PKLot':
+        loader = PKLotLoader()
+
+    if model_path is not None:
+        with open(model_path, 'rb') as fp:
+            model = pickle.load(fp)
+    elif model_type == 'MLP':
+        model = MLPClassifier(solver='lbfgs', hidden_layer_sizes=(15, 10))
```

Comment:
@rstevanak If we are talking about fixed number of layers you can pass `nargs` argument to click option which specifies how many values should command line argument have, for instance, if the nargs value is set to 2 click would expect in command line something like `--hidden_layers 20 10`. This would then return tuple of string `(u'20', u'10') where this can be easily converted to ints with map like `map(int, hidden_layers)`. If we are talking about configuring number of layers I would probably suggest using some different format where architecture of the model can be specified to its full extend but that might be slight overkill for a script.